duckduckgo/iOS
# update theme to use system colours

Task/Issue URL: https://app.asana.com/0/0/1206009605456322/f
Tech Design URL:
CC:

**Description**:

Rationalises some of the colours to use the system design and some minor UI fixes.

**Steps to test this PR**:
1. Launch the app on iPad and iPhone and test in Light and Dark mode
2. Smoke test the app (browsing, tabs, bookmarks, autofill, settings) and check that nothing jumps out as wrong
